firmware/firmware.h

```diff
@@ -65,7 +65,7 @@ void flash_disable();
 #define BRIDGE_USB 0
 #define BRIDGE_PORT_A 1
 #define BRIDGE_PORT_B 2
-#define BRIDGE_BUF_SIZE 256
+#define BRIDGE_BUF_SIZE 255
```

Comment:
:clap: I'm glad you caught this. I don't think it would have caused any issues yet because the bridge length field for transfers is only one byte (so at worst, we were waiting one bit of memory) but definitely good to have this cleaned up for whenever we build support for larger transfers.